pkgs/applications/science/math/weka/default.nix

```
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     mkdir -pv $out/share/weka
     cp -Rv * $out/share/weka
 
-    makeWrapper ${jre}/bin/java $out/bin/weka \
+    makeWrapper ${jdk11}/bin/java $out/bin/weka \
```

```suggestion
    makeWrapper ${jdk}/bin/java $out/bin/weka \
```

Parameterize the version in `all-packages.nix`.